src/Service/DeliveryManager.php

```diff
@@ -164,9 +164,12 @@ public function setDefaults(Delivery $delivery)
                     $duration = $this->routing->getDuration(...$coords);
 
                     $pickupDoneBefore = clone $dropoff->getDoneBefore();
+                    $pickupDoneAfter = clone $dropoff->getDoneAfter();
```

Comment:
The problem is that at this step, we may not have the beginning of the dropoff slot. 
On the API or on the form to create a delivery, the minimal data we ask is the end of the dropoff timeslot. 
The beginning of the slot, if missing, is then "calculated" with a default of 15 minutes in [TaskListener](https://github.com/coopcycle/coopcycle-web/blob/e247f64de5a5dccd2841372633d0ca6882162e2e/src/Entity/Listener/TaskListener.php#L31-L38)

Reply:
Got it, I'll check if it can be done another way